### PR TITLE
Update WireGuard Easy image in docker compose file.

### PR DESCRIPTION
**Pull request by thedroidkid**
The old image was not the official wireguard easy repo. 
https://github.com/wg-easy/wg-easy

### CODE REVIEW

**Comment by CorrectRoadH** [comment]
why new version docker remove tag after image name?e🤔

**Comment by malykhinvi** [comment]
Are there any blockers to merge it in?

**Comment by CorrectRoadH** [comment]
> Are there any blockers to merge it in?
We hope that docker compose has a specific image name and tag.🤔that will reduce the workload in check application upgradable.  like `ghcr.io/wg-easy/wg-easy:xxxx`

File: Apps/WireGuardEasy/docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: wg-easy
 services:
   wg-easy:
-    image: weejewel/wg-easy:7
+    image: ghcr.io/wg-easy/wg-easy
```
Comment (malykhinvi) [inline Apps/WireGuardEasy/docker-compose.yml:4]:
```suggestion
    image: ghcr.io/wg-easy/wg-easy:13
```
The latest version is 13: https://github.com/wg-easy/wg-easy/pkgs/container/wg-easy/217936898?tag=13

**Review by malykhinvi — APPROVED** [review]

File: Apps/WireGuardEasy/docker-compose.yml (hunk 1 above)
Comment (malykhinvi) [reply Apps/WireGuardEasy/docker-compose.yml:4]: @CorrectRoadH I've added version, please take a look ^^

Comment (CorrectRoadH) [reply Apps/WireGuardEasy/docker-compose.yml]: LGTM. Very Thanks for your contribution.

**Pull request merged**